identity/digital_twin.go

```diff
@@ -161,6 +161,7 @@ func (c *Client) getDigitalTwinWithProperties(ctx context.Context,
 }
 
 // GetDigitalTwin receive all properties for given digital twin.
+// Deprecated: Use the equivalent function in the knowledge package.
```

Comment:
Does this highlight deprecation in any way or is it just comments on the method?

Reply:
It's what the linter suggested, so I assume it will show up in a certain way in the go docs